account_statement_import_txt_xlsx/models/account_statement_import_sheet_parser.py

```diff
@@ -61,7 +62,9 @@ def parse(self, data_file, mapping, filename):
         lines = self._parse_lines(mapping, data_file, currency_code)
         if not lines:
             return currency_code, account_number, [{"transactions": []}]
-
+        if not isinstance(lines[0]["timestamp"], datetime):
```

Comment:
How this is related to empty balance column?

Reply:
Hello @alexey-pelykh , this line was necessary, because some banks' statements put the first line without a date, containing only the description and the previous balance, so the solution I found was to copy the date of the next line. But I'm open to suggestions, thanks for the review.

Reply:
This should be done as a separate PR. Backport is a backport, sneaky changes aren't good :)

Reply:
I understand. I will provide a new PR and fix this one.